docs/medical-api/api-reference/fhir/consolidated-data-query-post.mdx

```diff
@@ -69,11 +70,6 @@ Otherwise, the data will be returned in JSON format in the Webhook payload (defa
   [Medical Record Summary](/medical-api/handling-data/webhooks#medical-record) being sent.</Info>
 </ParamField>
 
-<Info>
-  You can scroll to the bottom of the page to view all [available
-  resources](/medical-api/api-reference/fhir/get-patient-consolidated#available-fhir-resources).
-</Info>
```

Comment:
Moved it to the respective `ParamField` as it helps reading the docs and finding the information.